releases/protocol/release-specs-v1.7.14-Interim.md

```diff
@@ -0,0 +1,124 @@
+[comment]: <> (tags/v1.7.16)
```

Comment:
Maybe we should pick a proper semver tag, 1.8 (new features), even if we've mistakenly used 1.7next1 until now?

Reply:
Indeed 1.8 would be the correct choice, considering that features were also added. But we somehow reserved 1.8.0 for Andromeda release, thus only increased the last number.